src/index.ts

```diff
@@ -1,111 +1,7 @@
 #!/usr/bin/env node
 
-import * as yargs from 'yargs';
-import setupHardRejection from 'hard-rejection';
-import reportFailure from './commands/report-failure';
-import merge from './commands/merge';
-import assert from './utils/assert';
+// This file is mainly for indirection w/the shebang so it's valid JavaScript
+// The real implementation is in ./run.ts
+import main from './run';
```

Comment:
This is slightly annoying but I don't know how to get the typescript langserver to be happy with a typescript file that has a shebang

Reply:
Seems fine (though somewhat annoying).

Reply:
@rwjblue if you can figure out how to do it i'm very happy to avoid the indirection